starboard/types.h

```diff
@@ -46,13 +46,7 @@ extern "C" {
 // --- Standard Include Emulation ----------------------------------------------
 
 #if !SB_HAS(SSIZE_T)
```

Comment:
Remove the SB_HAS_SSIZE_T macro, update the starboard/CHANGELOG.md  
https://source.corp.google.com/search?q=f:starboard%20SB_HAS_SSIZE_T&sq=r:lbshell-internal%2Fcobalt_src%20b:main